spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -136,9 +139,9 @@ public EurekaInstanceConfigBean eurekaInstanceConfigBean(InetUtils inetUtils) {
 			}
 			String scheme = instance.getSecurePortEnabled() ? "https" : "http";
 			instance.setStatusPageUrl(scheme + "://" + instance.getHostname() + ":"
-					+ this.managementPort + instance.getStatusPageUrlPath());
+					+ this.managementPort + this.managementContextPath + instance.getStatusPageUrlPath());
```

Comment:
Should only add this if not empty (ie not null and not empty). If it were null, `null` would go here.